src/freeze/enable.rs

```diff
@@ -0,0 +1,127 @@
+use super::*;
+
+pub struct EnableFreezeArgs {
+    pub keypair: Option<String>,
+    pub rpc_url: Option<String>,
+    pub cache: String,
+    pub config: String,
+    pub candy_machine: Option<String>,
+    pub freeze_days: Option<u8>,
+}
+
+pub fn process_enable_freeze(args: EnableFreezeArgs) -> Result<()> {
+    let sugar_config = sugar_setup(args.keypair.clone(), args.rpc_url.clone())?;
+    let client = setup_client(&sugar_config)?;
+    let program = client.program(CANDY_MACHINE_ID);
+    let config_data = get_config_data(&args.config)?;
+
+    // The candy machine id specified takes precedence over the one from the cache.
+    let candy_machine_id = match args.candy_machine {
+        Some(ref candy_machine_id) => candy_machine_id.to_owned(),
+        None => {
+            let cache = load_cache(&args.cache, false)?;
+            cache.program.candy_machine
+        }
+    };
+
+    let candy_pubkey = Pubkey::from_str(&candy_machine_id)
+        .map_err(|_| anyhow!("Failed to parse candy machine id: {}", &candy_machine_id))?;
+
+    // Freeze days specified takes precedence over the one from the config.
+    let freeze_time = if let Some(freeze_days) = args.freeze_days {
```

Comment:
Perhaps we should validate that `args.freeze_days` is in a valid range (`0 < freeze_days <= 30`):
<img width="682" alt="image" src="https://user-images.githubusercontent.com/729235/190193554-2764d19a-cfa4-473e-9a40-e61fe7d281d7.png">

Reply:
Should this take `freezeTime` in seconds instead? It seems like taking days is a better UI and advanced users can still set seconds in the config file. 

I'll add a check for `freeze_days > 30` which is the only range we need to validate given it's an unsigned integer.

Reply:
![Selection_963](https://user-images.githubusercontent.com/1684605/190214791-776de2f1-d6a3-43ee-8bcc-cb7fecf5363e.png)